overlay/lib/fetch.nix

```diff
@@ -1,9 +1,21 @@
 { buildPackages, lib }:
+let
+  unpackSrc = { name, src }: buildPackages.stdenvNoCC.mkDerivation {
+    inherit name src;
+    preferLocalBuild = true;
+    allowSubstitutes = false;
```

Comment:
Technically the `unpackPhase` could happen on a remote builder (I think) ;)

Reply:
> `allowSubstitutes`
> 
> If this attribute is set to false, then Nix will always build this derivation; it will not try to substitute its outputs. This is useful for very trivial derivations (such as writeText in Nixpkgs) that are cheaper to build than to substitute from a binary cache.

It's usually faster to just unpack locally instead of downloading from a cache so I set this to `false`.